ibm/multi_region.go

```diff
@@ -69,3 +108,81 @@ func getRegion(ctx context.Context, d *transform.TransformData) (interface{}, er
 	region := plugin.GetMatrixItem(ctx)["region"].(string)
 	return region, nil
 }
+
+// GetDefaultIBMRegion returns the default region for AWS partiton
+// if not set by Env variable or in aws profile
+func GetDefaultIBMRegion(d *plugin.QueryData) string {
+	// have we already created and cached the service?
+	serviceCacheKey := "GetDefaultIBMRegion"
+	if cachedData, ok := d.ConnectionManager.Cache.Get(serviceCacheKey); ok {
+		return cachedData.(string)
+	}
+	allIBMRegions := Regions()
+
+	// get aws config info
+	ibmConfig := GetConfig(d.Connection)
+
+	var regions []string
+	var region string
+
+	if ibmConfig.Regions != nil {
+		regions = ibmConfig.Regions
+		region = regions[0]
+	} else {
+		// Fetch regions from environment variables
+		if os.Getenv("IBMCLOUD_REGION") != "" {
+			region = os.Getenv("IBMCLOUD_REGION")
+		}
+		if os.Getenv("IC_REGION") != "" {
+			region = os.Getenv("IC_REGION")
+		}
+
+		if region != "" {
+			regions = []string{region}
+		}
+
+		if !helpers.StringSliceContains(allIBMRegions, region) {
+			regions = []string{"us-south"}
```

Comment:
Can you please add a comment on why we picked us-south with a link to doc if available?